templates/broker-deployment_template.yaml

```diff
@@ -18,16 +18,52 @@ spec:
     spec:
       restartPolicy: Always
       containers:
-        - image: "{{dockerhub_org}}/ansible-service-broker:latest"
+        - image: {{registry_org}}/ansible-service-broker:latest
```

Comment:
What if I want to pull ansible-service-broker from my org but get the apbs from ansibleplaybookbundle? I think re-using registry_org might make that hard?